source/_integrations/history_stats.markdown

```diff
@@ -178,6 +178,18 @@ Here, last Monday is _today_ as a timestamp, minus 86400 times the current weekd
 
 {% endraw %}
 
+**Next 4pm **: from 4pm today until 4pm tomorrow.
```

Comment:
```suggestion
**Next 4 pm**: 24 hours, from the last 4 pm till the next 4 pm. If it hasn't been 4 pm today, that would be 4 pm yesterday until 4 pm today. If it is already past 4 pm today, it will be 4 pm today until 4 pm tomorrow.
```

Reply:
Just wondering if it's worth mentioning the 8 hour buffer that would need changed if the time is changed?

Perhaps ... 

**Next 4 pm**: 24 hours, from the last 4 pm till the next 4 pm. If it hasn't been 4 pm today, that would be 4 pm yesterday until 4 pm today. If it is already past 4 pm today, it will be 4 pm today until 4 pm tomorrow. If changing the start time, then add or subtract to the 8 hour buffer.

Reply:
Not sure if I add this suggestion via a 'suggested change' I get "Authors can't do this" ...